docs/modules/operation/pages/topology/enlinkd/introduction.adoc

```diff
@@ -22,6 +22,34 @@ The following sections describe the Enlinkd daemon and its configuration.
 Additionally, the supported link discovery implementations are described as well as a list of the SNMP MIBs that the SNMP agents must expose for EnLinkd to gather links between nodes.
 You can view detailed information about a node's connections (discovered links) and supporting link data on the node detail page within the {page-component-title} Web UI.
 
+[[ga-enlinkd-layers]]
+== Topology Layers
+
+Enhanced Linkd Topology Map supports Layers protocol maps.
```

Comment:
```suggestion
The Enhanced Linkd topology map supports layered protocol maps.
```

Reply:
Could we add a reason why having the layers is useful?

Reply:
Well, this is a good point.

Topology Maps is a compelling point.

The idea behind having protocols maps is that you can have a better view removing some data.

LLDP and CDP are very similar protocols. By the way you can decide to look only on CDP network and only on LLDP network. This can give you an idea if the protocol is working properly and if is or not enabled on all devices.

OSPF maps are different, because you can be interested in inter routing connection.

Layers can be useful in many cases. You can view the network from different point of view and is what layers give you.

I know is generic but, in terms of use cases one could be:

I'm not interested in OSPF and IS-IS Link and I want just a map of Layer two links.
I'm only interested in looking at Layer 3 topology, so I do not want to see Layer 2 related links.

I'm only interested in bridge topology.